test/accreditation/generate_data.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+# pylint: disable= no-member
+"""
+Generate data for accreditation tests
+"""
+
+# Import general libraries (needed for functions)
+import pickle
+import qiskit
+
+
+# Import Qiskit classes
+from qiskit import QuantumCircuit, QuantumRegister, ClassicalRegister, execute
+from qiskit.providers.aer.noise import NoiseModel
+from qiskit.providers.aer.noise.errors.standard_errors import depolarizing_error
+
+# Import the accreditation functions.
+from qiskit.ignis.verification.accreditation import AccreditationFitter, AccreditationCircuits
+
+
+def make_accred_system():
+    """generate accred circuits"""
+    # Create a Quantum Register with n_qb qubits.
+    q_reg = QuantumRegister(4, 'q')
+    # Create a Classical Register with n_qb bits.
+    c_reg = ClassicalRegister(4, 's')
+    # Create a Quantum Circuit acting on the q register
+    target_circuit = QuantumCircuit(q_reg, c_reg)
+    target_circuit.h(0)
+    target_circuit.h(1)
+    target_circuit.h(2)
+    target_circuit.h(3)
+    target_circuit.cx(0, 1)
+    target_circuit.cx(0, 2)
+    target_circuit.h(1)
+    target_circuit.h(2)
+    target_circuit.cx(0, 3)
+    target_circuit.cx(1, 2)
+    target_circuit.h(1)
+    target_circuit.h(2)
+    target_circuit.h(3)
+    target_circuit.measure(q_reg, c_reg)
+
+    return AccreditationCircuits(target_circuit)
+
+
+def generate_data_ideal():
+    """generaete a ideal data pickle"""
+    accsys = make_accred_system()
+    simulator = qiskit.Aer.get_backend('qasm_simulator')
+
+    # Number of runs
+    d = 20
+    v = 10
+
+    all_results = []
+    all_postp_list = []
+    all_v_zero = []
+    for run in range(d):
+        print([run, d])
+        # Create target and trap circuits with random Pauli gates
+        circuit_list, postp_list, v_zero = accsys.generateCircuits(v)
+
+        job = execute(circuit_list,
+                      simulator,
+                      shots=1)
```

Comment:
```suggestion
                      shots=1, seed=seed_simulator + run)
```

Reply:
Are you sure this works? I get error 
```
Traceback (most recent call last):
  File "./generate_data.py", line 148, in <module>
    generate_data_noisy()
  File "./generate_data.py", line 122, in generate_data_noisy
    job = execute(circuit_list, simulator,
  File "/Users/sethmerkel/Research/Code/Repos/qiskit-terra/qiskit/execute.py", line 262, in execute
    return backend.run(qobj, **run_config)
TypeError: run() got an unexpected keyword argument 'seed'
```

from the line

`job = execute(circuit_list, simulator,
                      noise_model=noise_model, basis_gates=basis_gates,
                      shots=1, seed=seed_simulator + run)`

Simulator is qasm simulator

Reply:
Sorry, I had a typo: its `seed_simulator=seed_simulator + run`